sn_node/tests/nodes_rewards.rs

```diff
@@ -198,13 +212,17 @@ fn spawn_transfer_notifs_listener(endpoint: String) -> JoinHandle<Result<usize,
 
         let mut count = 0;
         let mut stream = response.into_inner();
+        let royalties_pk = NETWORK_ROYALTIES_PK.public_key();
+
         while let Some(Ok(e)) = stream.next().await {
             match NodeEvent::from_bytes(&e.event) {
-                Ok(NodeEvent::TransferNotif { key, transfer: _ }) => {
+                Ok(NodeEvent::TransferNotif { key, cash_notes: _ }) => {
                     println!("Transfer notif received for key {key:?}");
-                    count += 1;
-                    if count == CLOSE_GROUP_SIZE {
-                        break;
+                    if key == royalties_pk {
```

Comment:
will there still be normal transfer (i.e. not targeting to network_royalty) ?
if so, this will result in an infinite wait ?

Reply:
not at the moment, if there is then the test shall fail with a timeout